### PR TITLE
POL-449 Prepare policies for chrome v2

### PR DESCRIPTION
**Pull request by josejulio**
To merge with: https://github.com/RedHatInsights/policies-ui-frontend/pull/232

### CODE REVIEW

**Comment by karelhala** [comment]
@josejulio we can merge this before you merge the UI, but once the UI has been merged this one has to be in. It's your call either merge it now or right before the UI PR is merged, what would you prefer?

**Comment by josejulio** [comment]
> @josejulio we can merge this before you merge the UI, but once the UI has been merged this one has to be in. It's your call either merge it now or right before the UI PR is merged, what would you prefer?

I'm +1 on mergin this now

**Review by kruai — APPROVED** [review]

**Pull request merged**
